readme.md

```diff
@@ -107,7 +128,7 @@ plugin is able to draw Swagger-template to display the generated documentation f
  - **@summary** - short description of request
  - **@description** - Implementation Notes
  - **@_204** - Custom description of response code. You can specify any code as you want.
- - **@some_field** - Description of the field from the rules method
+ - **@first_name** - Description of the field from the rules method
  
  If you do not create a class Request, the summary, Implementation Notes and parameters will be empty. 
```

Comment:
please remove the text below to the line `so, you can specify way to collect documentation by creating your custom data collector class.`